item.rb

```diff
@@ -0,0 +1,41 @@
+require 'date'
+class Item
+  attr_accessor :source, :publish_date, :self, :label
```

Comment:
- [ ] Nice job addressing the comment from the previous code review, however, in addressing the comment we left out the `:label`. Let us kindly fix it